eth_account/messages.py

```diff
@@ -14,20 +14,30 @@
 )
 
 
-def defunct_hash_message(primitive=None, hexstr=None, text=None):
+def defunct_hash_message(
+        primitive=None,
+        *,
+        hexstr=None,
+        text=None,
+        signature_version=b'E',
+        version_specific_data=None):
     '''
     Convert the provided message into a message hash, to be signed.
     This provides the same prefix and hashing approach as
-    :meth:`w3.eth.sign() <web3.eth.Eth.sign>`. That means that the
-    message will automatically be prepended with text
-    defined in EIP-191 as version 'E': ``b'\\x19Ethereum Signed Message:\\n'``
+    :meth:`w3.eth.sign() <web3.eth.Eth.sign>`.
+    Currently you can only specify the ``signature_version`` as following.
+    version ``0x45`` (version ``E``): ``b'\\x19Ethereum Signed Message:\\n'``
     concatenated with the number of bytes in the message.
+    NOTE: This is the defualt version followed, if the signature_version is not specified.
```

Comment:
To get this to render properly in the docs this should be updated to use the following syntax.

```rst
.. note: This is the default version ...
```